deptofdefense/move.mil
# Change homepage banner svg to jpg

## Checklist

I have…


## Summary of Changes

This pull request…

Swaps out the SVG for a JPEG. It's slightly lower quality but it's not a huge difference.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo
2. git checkout hero_art
3. set up development dependencies according to CONTRIBUTING.md,
4. run bin/rails server, and
5. load up http://localhost:3000 in your Web browser of choice

## Screenshots

<img width="1680" alt="screen shot 2017-11-02 at 2 49 28 pm" src="https://user-images.githubusercontent.com/29409348/32344382-11fd925a-bfdd-11e7-9adc-19a0acb86da8.png">
